.drone.yml

```diff
@@ -106,6 +63,89 @@ steps:
     password:
       from_secret: docker_password
 
+---
+kind: pipeline
+name: test
+
+depends_on:
+- publish to docker hub
+
+steps:
+- name: download testdata
+  image: alpine/git:1.0.7
+  volumes:
+  - name: test-collections-v
+    path: /drone/src/testsuite/volumes/collections/testdata
+  commands:
+  - mkdir -p testsuite/volumes/collections
+  - git clone https://github.com/hoover/testdata testsuite/volumes/collections/testdata
+  - chown -R 666:666 testsuite/volumes/collections/testdata
```

Comment:
we shouldn't need this - default permissions for this volume should be that everyone can read the files. This is the case in production, too: the initial collection data will remain owned by `root:root`